### ISSUE TITLE
Incorporate accessibility recommendations into product

### ISSUE THREAD

**Issue by amysantee** [hackoregon/openelections#433]
Review and incorporate accessibility feedback as needed. Report is located in Elections team Design folder.

**Comment by andrewbiang888** [comment]
@jaronheard Having trouble locating this doc. Could you send it my way when you have a minute?

**Comment by jaronheard** [comment]
@andrewbiang888 https://drive.google.com/open?id=1VwDvX3lXmsb9Y2csB2S24CG5e-MZ1r3d

**Comment by andrewbiang888** [comment]
Sweet. Putting together a checklist for myself from the document:
- [ ] Get pink (or contrasted font color) to meed contrast standards of at least WCAG AA
<img width="342" alt="Screen Shot 2020-04-19 at 1 33 33 PM" src="https://user-images.githubusercontent.com/20327807/79699142-601a2500-8242-11ea-96b1-6c204a338678.png">

- [ ] Red does not meet contrast obligations, Recommendation: Change to green button color (or make text dark civic purple, which passes AA standards)

<img width="266" alt="Screen Shot 2020-04-19 at 1 34 43 PM" src="https://user-images.githubusercontent.com/20327807/79699165-88a21f00-8242-11ea-8574-564713a3a4ee.png">

- [ ] Grey does not meet contrast obligations (change to #757575)

<img width="353" alt="Screen Shot 2020-04-19 at 1 38 21 PM" src="https://user-images.githubusercontent.com/20327807/79699236-0b2ade80-8243-11ea-8474-e2147e3d6dc0.png">

- [ ] Green does not meet contrast obligations. Recommendation: Change the green button color (or make text dark civic purple, which passes AA standards)
<img width="326" alt="Screen Shot 2020-04-19 at 1 43 30 PM" src="https://user-images.githubusercontent.com/20327807/79699323-c2bff080-8243-11ea-9ac0-c9f504fc2ce5.png">

- [ ] Recent Activity - humanize information by including donor’s name (not who uploaded it) and the amount donated.
<img width="238" alt="Screen Shot 2020-04-19 at 1 44 59 PM" src="https://user-images.githubusercontent.com/20327807/79699354-f7cc4300-8243-11ea-8231-46030b4f25e0.png">

- [ ] On dashboard, move status box to top.

- [ ] Additional Aria updates (reference doc linked above).
